src/solver/conv_wino_fury_RxS.cpp

```diff
@@ -0,0 +1,569 @@
+                !is_backWrW ? primitive_params.CastTo<conv::DataInvokeParams>().tensors.out
+                            : primitive_params.CastTo<conv::WrWInvokeParams>().tensors.dw;
+
+            // clang-format off
+            MIOPEN_LOG_I2(
+              "\nn_groups=" << conv.n_groups <<
+              "\nout_Hs=" << conv.out_Hs <<
+              "\nout_We=" << conv.out_We <<
+              "\nd_W_window=" << conv.d_W_window <<
+              "\nd_H_window=" << conv.d_H_window <<
+              "\nd_W_clip_bot_neg=" << conv.d_W_clip_bot_neg <<
+              "\nd_H_clip_bot_neg=" << conv.d_H_clip_bot_neg <<
+              "\nd_strides.nk=" << d_strides.nk <<
+              "\nd_strides.h=" << d_strides.h <<
+              "\nd_strides.c=" << d_strides.c <<
+              "\no_W_window=" << conv.o_W_window <<
+              "\no_H_window=" << conv.o_H_window <<
+              "\no_W_clip_bot_neg=" << conv.o_W_clip_bot_neg <<
+              "\no_H_clip_bot_neg=" << conv.o_H_clip_bot_neg <<
+              "\no_strides.nk=" << o_strides.nk <<
+              "\no_strides.h=" << o_strides.h <<
+              "\no_strides.c=" << o_strides.c <<
+              "\nN=" << conv.N <<
+              "\nK=" << conv.K <<
+              "\nC=" << conv.C <<
+              "\nR=" << conv.R <<
+              "\nS=" << conv.S <<
+              "\nf_strides.nk=" << f_strides.nk <<
+              "\nf_strides.c=" << f_strides.c <<
+              "\nf_strides.h=" << f_strides.h <<
+              "\nf_strides.w=" << f_strides.w <<
+              "\nf_RS_offset=" << f_RS_offset <<
+              "\nflags=" << flags <<
+              "\nactivation_mode=" << static_cast<uint32_t>(activation_mode) <<
+              "\n");
+            // clang-format on
+
+            k(conv.n_groups,
+              conv.out_Hs,
+              conv.out_We,
+              conv.d_W_window,
+              conv.d_H_window,
+              conv.d_W_clip_bot_neg,
+              conv.d_H_clip_bot_neg,
+              uint32_t(d_strides.nk),
+              uint32_t(d_strides.h), // todo: check < 2^16
+              uint32_t(d_strides.c),
+              conv.o_W_window,
+              conv.o_H_window,
+              conv.o_W_clip_bot_neg,
+              conv.o_H_clip_bot_neg,
+              uint32_t(o_strides.nk),
+              uint32_t(o_strides.h), // todo: check < 2^16
+              uint32_t(o_strides.c),
+              data_tensors,
+              out_tensors,
+              filter_tensors,
+              conv.N,
+              conv.K,
+              conv.C,
+              conv.R,
+              conv.S,
+              uint32_t(f_strides.nk),
+              uint32_t(f_strides.c),
+              int32_t(f_strides.h),
+              int32_t(f_strides.w),
+              f_RS_offset,
+              uint64_t(0),
+              nullptr,
+              flags,
+              activation_mode,
+              uint8_t(0),
+              uint16_t(0),
+              0.0);
```

Comment:
Be careful, `0.0` is `double`
```suggestion
              0.0f);
```

Reply:
Already fixed in local version